exercises/hangman/src/main/java/Output.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * (C) ActiveViam 2019
+ * ALL RIGHTS RESERVED. This material is the CONFIDENTIAL and PROPRIETARY
+ * property of ActiveViam. Any unauthorized use
+ * reproduction or transfer of this material is strictly prohibited
+ */
+
+import java.util.List;
+import java.util.Set;
+
+class Output {
+
```

Comment:
Did you add this file so that students of the track have an easier start with this exercise? I think it is a good idea, this FRP is not well known or used in daily coding.
Maybe it would also be good to add stubs to Hangman.java.